src/components/dropdown/_switcher/dropdown_switcher_link.bemhtml

```diff
@@ -0,0 +1,28 @@
+block('dropdown').mod('switcher', 'link').elem('switcher').def()(function() {
+
+    // FIXME: it's should work fine!
+    return applyNext();
+
+    const React = require('react');
+    const Link = require('../src/components/link/link');
+    const dropdown = this._dropdown;
+    const switcher = dropdown.switcher;
+
+    if(Array.isArray(switcher)) return switcher;
+
+    let dropdownMods = this.mods;
+    return this.isSimple(switcher) ?
+        applyCtx(<Link
```

Comment:
@nik-kor it doesn't work. The first attrs it object but react waits string or class. Can you check applyCtx?